### PR TITLE
include force-dynamic in Next.js docs

### LINKED ISSUE #5619: Expand Next.js specific documentation to include force-dynamic
Following from this discussion: https://github.com/prisma/prisma/discussions/22934#discussioncomment-8374081

It appears that our documentation is not sufficient and routes that use Prisma client may require the [Next.js `dynamic` option](https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#dynamic) to be set to `'force-dynamic'` in order to function.

### PR DESCRIPTION
**Pull request by jharrell**
Fixes #5619

### CODE REVIEW

File: content/200-orm/800-more/600-help-and-troubleshooting/100-help-articles/400-nextjs-prisma-client-dev-practices.mdx [hunk 1]
```diff
@@ -71,3 +71,20 @@ export const getServerSideProps = async () => {
   return { props: { posts } }
 }
 ```
+
+## Next.js caching
+
+When using Next.js, you may run into an issue where [data is not updated and becomes stale in certain routes](https://github.com/prisma/prisma/discussions/22934#discussioncomment-8374081). In these cases it's important to [set the dynamic behavior of the route](https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#dynamic) to `"force-dynamic"` so data is correctly returned from your Prisma queries.
```
Comment (ankur-arch) [inline content/200-orm/800-more/600-help-and-troubleshooting/100-help-articles/400-nextjs-prisma-client-dev-practices.mdx]:
```suggestion
When using Next.js, you may run into an issue where [only stale data is returned from certain routes](https://github.com/prisma/prisma/discussions/22934#discussioncomment-8374081). In these cases, it's important to [set the dynamic behavior of the route](https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#dynamic) to `"force-dynamic"` so data is correctly returned from your Prisma queries.
```

File: content/200-orm/800-more/600-help-and-troubleshooting/100-help-articles/400-nextjs-prisma-client-dev-practices.mdx [hunk 2]
```diff
@@ -71,3 +71,20 @@ export const getServerSideProps = async () => {
...
+
+In a simple example, you can add `"force-dynamic"` like this:
```
Comment (ankur-arch) [inline content/200-orm/800-more/600-help-and-troubleshooting/100-help-articles/400-nextjs-prisma-client-dev-practices.mdx]:
```suggestion
You can add `"force-dynamic"` like this:
```

File: content/200-orm/800-more/600-help-and-troubleshooting/100-help-articles/400-nextjs-prisma-client-dev-practices.mdx [hunk 3]
```diff
@@ -71,3 +71,20 @@ export const getServerSideProps = async () => {
   return { props: { posts } }
 }
 ```
+
+## Next.js caching
+
+When using Next.js, you may run into an issue where [only stale data is returned from certain routes](https://github.com/prisma/prisma/discussions/22934#discussioncomment-8374081). In these cases, it's important to [set the dynamic behavior of the route](https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#dynamic) to `"force-dynamic"` so data is correctly returned from your Prisma queries.
```
Comment (grpatel) [inline content/200-orm/800-more/600-help-and-troubleshooting/100-help-articles/400-nextjs-prisma-client-dev-practices.mdx]:
```suggestion
When using Next.js, you may run into an issue where [only stale data is returned from certain routes](https://github.com/prisma/prisma/discussions/22934#discussioncomment-8374081). In these cases, it's important to [set the dynamic behavior of the route](https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#dynamic) to `"force-dynamic"` so data is correctly returned from your Prisma Client queries.
```

Just a nitpicky thing

**Review by grpatel — COMMENTED** [review]
Looks great

**Review by mhessdev — APPROVED** [review]
Looks good to me!

**Review by janpio — COMMENTED** [review]
1. The headline levels of this page are now inconsistent. (Problem and Solution are not the same level as the new "Next.js caching")
2. With the removal of "instantiating" from the page title, the actual issue that the Problem and Solution headlines refer to is not mentioned any more at all.

Suggestion: Change Problem and Solution to `###` and add another `##` above that spells out the issue these are about again.

**Pull request merged**